sdl2promela/sdl/model.py

```diff
@@ -1183,6 +1204,18 @@ def convert(source: ogAST.PrimIndex):
     return result
 
 
+@dispatch(ogAST.PrimConditional)
+def convert(source: ogAST.PrimConditional):
+    result = ConditionalExpression()
+
+    result.condition = convert(source.value["if"])
+    result.trueExpression = convert(source.value["then"])
+    result.falseExpression = convert(source.value["else"])
+
+    result.type = source.exprType
```

Comment:
Type is not defined in the ConditionalExpression....

Reply:
It is, base class `Expression`